docs/_docs/focus/convert.md

```diff
@@ -57,35 +62,33 @@ The following mapping is assuming you have all amortized cost rows and only comm
 | ListUnitPrice              | EA: PayGPrice<br>MCA: PayGPrice \* ExchangeRate     | None                                                                                                                                                                  |
 | PricingCategory            | PricingModel                                        | If "OnDemand", then `Standard`; if "Spot", then `Dynamic`; if "Reservation" or "Savings Plan", then `Committed`; otherwise, null                                      |
-| PricingQuantity            | Quantity                                            | Map UnitOfMeasure using [Pricing units data file](../../_reporting/data/README.md#-pricing-units) and divide Quantity by the PricingBlockSize<sup>2</sup>             |
-| PricingUnit                | UnitOfMeasure                                       | Map using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)                                                                                   |
+| PricingQuantity            | Quantity / x_PricingBlockSize (lookup)              | Map UnitOfMeasure to x_PricingBlockSize using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)<sup>1</sup>                                   |
+| PricingUnit                | DistinctUnits (lookup)                              | Map UnitOfMeasure to DistinctUnits using [Pricing units data file](../../_reporting/data/README.md#-pricing-units)                                                    |
 | ProviderName               | `Microsoft`                                         | None                                                                                                                                                                  |
 | PublisherName              | PublisherName                                       | None                                                                                                                                                                  |
 | RegionId                   | focus:RegionName                                    | Lowercase and remove spaces                                                                                                                                           |
-| RegionName                 | ResourceLocation                                    | Map using [Regions data file](../../_reporting/data/README.md#-regions)<sup>3</sup>                                                                                   |
+| RegionName                 | ResourceLocation                                    | Map ResourceLocation to RegionName using [Regions data file](../../_reporting/data/README.md#-regions)<sup>2</sup>                                                    |
```

Comment:
Clarify that the `OriginalValue` column in the reference data file is what is used to map from `ResourceLocation`.

Reply:
Good catch! Do you think this is clear enough? 🤔
```suggestion
| RegionName                 | ResourceLocation                                    | Map ResourceLocation (OriginalValue) to RegionName using [Regions data file](../../_reporting/data/README.md#-regions)<sup>2</sup> |
```

Reply:
Yes, I believe this works just fine.